app/server/apps/users/routes.js

```diff
@@ -1,28 +1,12 @@
 'use strict';
 var auth = require('../../middleware/authenticate'),
-    users = require('../../middleware/users'),
-    permissions = require('../../middleware/permissions'),
-    send = require('../../middleware/send');
+    routeGenerator = require('../../route-generator'),
+    methods = require('./methods'),
+    middlewares = require('./middlewares');
+
 module.exports = {
     pre: {
         all: [auth.isAuthenticated]
     },
-    routes: {
-        '/users': {
```

Comment:
I see, you're using the latest jefferson version, but you needed method names to attach to routes